lib/sprockets/commoner/processor.rb

```diff
@@ -47,6 +47,15 @@ def self.call(input)
         instance(input[:environment]).call(input)
       end
 
+      def self.configure(env, *args, **kwargs)
```

Comment:
Do we need to pass env here? Is it a more reusable pattern to do it this way? We could just call `Rails.application.config.assets.configure` within this configure block to get the env but that only works for a rails environment. I am not sure if this gem is designed to be used outside of rails or not.